media-api/conf/routes

```diff
@@ -14,6 +14,7 @@ GET     /images/aggregations/date/:field              controllers.AggregationCon
 # Images
 GET     /images/:id                                   controllers.MediaApi.getImage(id: String)
 GET     /images/:id/_elasticsearch                    controllers.MediaApi.getImageFromElasticSearch(id: String)
+GET     /images/:id/diffProjection                    controllers.MediaApi.diffProjection(id: String)
```

Comment:
Can we use something like `/images/:id/projection/diff` so we can group the different projection endpoints together?

Reply:
Will do.